src/main/java/pl/javastart/task/logic/CardPhoneContract.java

```diff
@@ -0,0 +1,77 @@
+package pl.javastart.task.logic;
+
+public class CardPhoneContract extends Contract {
+    double accountBalance;
+    int sumPhoneCallSeconds;
+    static final int SECONDS_IN_MINUTE = 60;
+
+    public CardPhoneContract(double smsCost, double mmsCost, double oneMinuteCallCost, double balance) {
+        super(smsCost, oneMinuteCallCost, mmsCost);
+        this.accountBalance = balance;
+    }
+
+    public double getAccountBalance() {
+        return accountBalance;
+    }
+
+    void setAccountBalance(double accountBalance) {
+        this.accountBalance = accountBalance;
+    }
+
+    @Override
+    public void sendSms() {
+        if (accountBalance == 0 || accountBalance < getSmsCost()) {
+            noMoneyToSendSms();
+        } else if (accountBalance >= getSmsCost()) {
+            smsSent();
+            setAccountBalance(getAccountBalance() - getSmsCost());
+            sumSentSms();
+        }
+    }
+
+    @Override
+    public void sendMms() {
+        if (accountBalance == 0 || accountBalance < getMmsCost()) {
+            noMoneyToSendMms();
+        } else if (accountBalance >= getMmsCost()) {
+            mmsSent();
+            setAccountBalance(getAccountBalance() - getMmsCost());
+            sumSentMms();
+        }
+    }
+
+    @Override
+    public void call(int seconds) {
+        if (accountBalance == 0 && (accountBalance * 100) < seconds) {
+            noMoneyToCall();
+        } else if ((accountBalance * 100) >= seconds) {
```

Comment:
Nie rozumiem co oznacza wartość 100. Skoro porównujesz to z sekundami to może powinno być 60? Chociaż nawet jeżeli tak, to nadal nie do końca rozumiem dlaczego stan konta wyrażony np. w zł porównujesz z sekundami, które są jednostką czasu